### ISSUE TITLE
[A11y]Tooltip provided for blue checkmark is not available for keyboard only user.

### ISSUE THREAD

**Issue by ryuyu** [NuGet/NuGetGallery#8490]
Link: https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1294859

## ​Environment Details:
Application Name: NuGet Service
#URL: https://www.nuget.org
Operating System: Windows 10
Browser Version: Chrome Version 85.0.4183.121 (Official Build) (64-bit),
Chromium Edge Version 87.0.637.0 (Official build) dev (64-bit)

## Repro Steps:​
 Hit the URL " https://www.nuget.org" and login with appropriate credentials.
Tab Navigate to "Sign in" control and hit enter.
 Tab Navigate to "Sign in with Microsoft" control and hit enter.
Search for a key term (say “json”) from search box.
"Packages returned for json" screen gets opened.
Tab navigate to Correct icon and observe its actionable using keyboard or not.
## Actual:​
Tooltip provided for icon (\/) besides Newtonsoft.Json link is not provided for keyboard user.
## Expected:​
Tooltip provided for icon (\/) besides Newtonsoft.Json link is should be provided for keyboard user.
## ​User Impact:​
If tooltip is not provided for keyboard user then keyboard user will miss some information that provided in the tooltip.

### Notes
The tooltip information that displays when hovering the blue checkmark on the search results page contains information that is not surface-able in any other way. We need to provide a text alternative to the hover text. UI will likely need to be discussed.
